tests/memberclient.py

```diff
@@ -55,7 +55,12 @@ def run(args):
         assert proposal_id == 0
 
```

Comment:
Can we add tests for all the new error cases introduced? Not being able to vote on an open proposal, not being able to withdraw a proposal that isn't open, trying to withdraw someone else's proposal and failing.

Reply:
I've added more tests, and I believe we cover everything now:
- Accepted votes won't process more votes
- Accepted proposal can't be withdrawn
- Non-proposers can't withdraw or remove proposal
- Withdrawn proposals can't be withdrawn again
- Withdrawn proposals won't process votes
- Removed proposals can't be withdrawn
- Removed proposals won't process votes
- Removed proposals can't be removed again